重构CellData

Nawodous opened the pull request.

Nawodous committed:
重构CellData

MilkMangoTea reviewed (dismissed):
**Data?**

NovaShen555 reviewed (changes requested):
将类型定义全丢struct文件夹里, 使用manager管理而非static方法

Nawodous committed:
啊啊啊啊啊宝宝你是一个一个一个
将UnitData和LandformData分别移动至struct文件夹下的UnitData.cs和LanformData.cs。
重写了UnitManager和LandformManager，分别保存在Manager文件夹下的UnitManager.cs和LandformManager.cs。
修改了CellData.cs以适配上述修正。

NovaShen555 reviewed (approved):
approve
